net.puddletag.puddletag.yaml

```diff
@@ -0,0 +1,94 @@
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: 5.15-21.08
+command: puddletag
+rename-desktop-file: puddletag.desktop
+rename-icon: puddletag
+finish-args:
+  - --device=dri
+  - --filesystem=xdg-download
+  - --filesystem=xdg-music
+  - --filesystem=xdg-pictures
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+build-options:
+  env:
+    - BASEAPP_REMOVE_WEBENGINE=1
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+
+modules:
+  - name: python3-configobj
+    buildsystem: simple
+    build-commands:
+      - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} "configobj" --no-build-isolation
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/64/61/079eb60459c44929e684fa7d9e2fdca403f67d64dd9dbac27296be2e0fab/configobj-5.0.6.tar.gz
+        sha256: a2f5650770e1c87fb335af19a9b7eb73fc05ccf22144eb68db7d00cd2bcb0902
+        x-checker-data:
+          type: pypi
+          name: configobj
+
+  - name: python3-mutagen
+    buildsystem: simple
+    build-commands:
+      - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}"
+        --prefix=${FLATPAK_DEST} "mutagen" --no-build-isolation
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/f3/d9/2232a4cb9a98e2d2501f7e58d193bc49c956ef23756d7423ba1bd87e386d/mutagen-1.45.1.tar.gz
+        sha256: 6397602efb3c2d7baebd2166ed85731ae1c1d475abca22090b7141ff5034b3e1
+        x-checker-data:
+          type: pypi
+          name: mutagen
+
+  - name: python3-pillow
+    buildsystem: simple
+    build-commands:
+      - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} "pillow" --no-build-isolation
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/43/6e/59853546226ee6200f9ba6e574d11604b60ad0754d2cbd1c8f3246b70418/Pillow-9.1.1.tar.gz
+        sha256: 7502539939b53d7565f3d11d87c78e7ec900d3c72945d4ee0e2f250d598309a0
+        x-checker-data:
+          type: pypi
+          name: pillow
+
+  - name: puddletag
+    buildsystem: simple
+    build-commands:
+      - python3 setup.py build
+      - python3 setup.py install --skip-build --prefix=/app --root=/ --optimize=1
+      - install -Dm644 /app/share/{pixmaps,icons/hicolor/256x256/apps}/puddletag.png
+      - rm -r /app/share/pixmaps
+    sources:
+      - type: archive
+        url: https://github.com/puddletag/puddletag/archive/refs/tags/2.1.1.tar.gz
+        sha256: 4ce85a78c49c8cf6b15f2c52c93374ae875b9aa83b529ede68547b68e9b85066
+        x-checker-data:
+          type: anitya
+          project-id: 3727
+          stable-only: true
+          url-template: https://github.com/puddletag/puddletag/releases/download/$version/puddletag-$version.tar.gz
+      - type: file
+        path: net.puddletag.puddletag.appdata.xml
```

Comment:
it's never installed. That's why the build fails

Reply:
Added the metainfo installation step.